pkg/provisioner/templates/container-toolkit.go

```diff
@@ -37,22 +37,32 @@ curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dear
 sudo apt-get install -y nvidia-container-toolkit
 
 # Configure container runtime
-sudo nvidia-ctk runtime configure --runtime={{.ContainerRuntime}} --set-as-default
+sudo nvidia-ctk runtime configure --runtime={{.ContainerRuntime}} --set-as-default --cdi.enable={{.EnableCDI}}
```

Comment:
I think `--enable-cdi` reads better.
```suggestion
sudo nvidia-ctk runtime configure --runtime={{.ContainerRuntime}} --set-as-default --enable-cdi={{.EnableCDI}}
```

Reply:
Not a blocker.

Reply:
Did you update this?

Reply:
Fixed